addons/mail/static/src/core/web/activity_mail_template.js

```diff
@@ -54,10 +59,14 @@ export class ActivityMailTemplate extends Component {
         ev.stopPropagation();
         ev.preventDefault();
         this.props.onClickButtons();
+        const thread = this.threadService.getThread(
```

Comment:
should not use `getThread`

Reply:
I feel like the name of the function is really confusing. It is too general but actually it is only for `chatter` and specifically for init in `chatter`

Reply:
Yes I agree, in master we should remove it completely. Alex might do that as part of cleaning up PRs actually?